examples/flutter_gallery/web/index.html

```diff
@@ -18,6 +18,16 @@
   <link rel="manifest" href="/manifest.json">
 </head>
 <body>
+  <!-- This script installs service_worker.js to provide PWA functionality to
+       application. For more information, see:
+       https://developers.google.com/web/fundamentals/primers/service-workers -->
+    <script>
```

Comment:
indent -2 spaces

Reply:
Fixed

Reply:
As `flutter_service_worker.js` fetches from cache first of all, it doesn't detect new deployed version of my app.

```
self.addEventListener('fetch', function (event) {
  event.respondWith(
    caches.match(event.request)
      .then(function (response) {
        if (response) {
          return response;
        }
        return fetch(event.request);
      })
  );
});
```

Do you guys have anything in mind to allow different fetch strategies?